### PR TITLE
Set redis password in slurm deployment documentation.

### PR DESCRIPTION
**Pull request by robertnishihara**
We need to set the redis password when deploying on slurm.

### CODE REVIEW

**Comment by robertnishihara** [comment]
@dpfried and @mitchellstern got this working on the Savio cluster at Berkeley.

**Comment by robertnishihara** [comment]
@gregSchwartz18 thanks for writing the documentation explaining how to use Ray in Slurm in https://github.com/ray-project/ray/pull/5467. I just used this today, and it was a huge help.

File: doc/source/deploying-on-slurm.rst [hunk 1]
```diff
@@ -26,38 +26,45 @@ Clusters managed by Slurm may require that Ray is initialized as a part of the s
   ip_prefix=$(srun --nodes=1 --ntasks=1 -w $node1 hostname --ip-address) # Making address
   suffix=':6379'
   ip_head=$ip_prefix$suffix
+  redis_password=$(python -u -c "import random; print(random.randrange(10**12))")
```
Comment (richardliaw) [inline doc/source/deploying-on-slurm.rst]: nit - use uuid instead?

**Review by richardliaw — APPROVED** [review]

File: doc/source/deploying-on-slurm.rst (hunk 1 above)
Comment (mitchellstern) [reply doc/source/deploying-on-slurm.rst]: The command `uuidgen` works both on my MacBook Pro as well as on the Berkeley Savio cluster.

Example:
```
$ uuidgen
73B96B27-93A9-4CF2-B137-590EE59DEC66
```

**Comment by robertnishihara** [comment]
@mitchellstern @dpfried, would one of you be able to try out these new instructions and script on the Savio cluster before (or after) merging this PR?

**Comment by dpfried** [comment]
I verified that the new script and instructions work on Savio. Thanks!

**Comment by robertnishihara** [comment]
Thanks @dpfried!

**Pull request merged**